src/cocaine/burlak/defaults.py

```diff
@@ -47,6 +47,7 @@ class Defaults(object):
     APPS_POLL_INTERVAL_SEC = 60
     INPUT_QUEUE_SIZE = 1024
 
+    CONTROL_RETRY_ATTEMPTS = 65000
```

Comment:
It's too huge number, in current architecture, single and only control coroutine would be locked for CONTROL_RETY_ATTEMPS *  DEFAULT_RETRY_EXP_BASE_SEC seconds in case of control exception, which means "forever" )), lets make defaults 4 or 3.